skt/executable.py

```diff
@@ -540,6 +540,15 @@ def setup_parser():
         type=str,
         help="Base repo URL"
     )
+    parser_merge.add_argument(
+        "--extra-git-args",
+        type=int,
```

Comment:
The docs suggest passing a string to the option, but here it says `type=int`. Based on what the option should do, I believe the docs are right ;)

Reply:
Oops! That was from a previous revision. I'll test that, too.